FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiService.kt

```diff
@@ -56,7 +56,7 @@ interface ApiService {
   ): Response<WelcomeMessageResponse>
 
   /**
-   * This method create a [Call] object for the API "https://flowcrypt.com/api/help/feedback"
+   * This method create a [Call] object for the API "https://flowcrypt.com/shared-tenant-fes/help/feedback"
```

Comment:
@tomholub Is this change wrong too? right?

Reply:
/help/feedback should remain original for now